Eipip 25 meeting notes (#51)
ethereum-cat-herders/EIPIP: EIPIP Meeting 25 Notes.md

# EIPIP Meeting 25 Notes
## Meeting Date/Time: Wednesday 27 January at 15:00 UTC
### Meeting Duration: 1 hour
### [GitHub Agenda Page](https://github.com/ethereum-cat-herders/EIPIP/issues/48)
### [Audio/Video of the meeting](https://www.youtube.com/watch?v=zz1IhYKztuc)
### Moderator: Pooja Ranjan
### Notes: Shane Lightowler

----
## DECISION ITEMS

> **DECISION**: The new Cat Herders website should focus on being an index/intro rather than an in-depth content repository. It should point to canonical sources elsewhere (eg Eth1.0 specs repo, Ethereum.org) to avoid duplication [16:03](https://youtu.be/zz1IhYKztuc?t=963)

Reviewer: ```suggestion
**DECISION 25.1**: The new Cat Herders website should focus on being an index/intro rather than an in-depth content repository. It should point to canonical sources elsewhere (eg Eth1.0 specs repo, Ethereum.org) to avoid duplication [16:03](https://youtu.be/zz1IhYKztuc?t=963)
```

> **DECISION**: Consensus that offering a gitcoin bounty of 7000USD for 1 months work for a contractor to move the merge bot to GitHub actions is a good idea. This to be followed up on next Cat Herders call. [29:43](https://youtu.be/zz1IhYKztuc?t=1783)

Reviewer: ```suggestion
**DECISION 25.2**: Consensus that offering a gitcoin bounty of 7000USD for 1 month of work for a contractor to move the merge bot to GitHub actions is a good idea. This to be followed up on the next Cat Herders call. [29:43](https://youtu.be/zz1IhYKztuc?t=1783)
```

## ACTION ITEMS

> **ACTION**: Confirm with Sam @ ethereum.org whether the intention is to have ethereum.org point to the Eth1.0 specs repo, or vice versa. Where does the canonical content live? [6:14](https://youtu.be/zz1IhYKztuc?t=374)

Reviewer: ```suggestion
**ACTION 25.1**: Confirm with Sam @ ethereum.org whether the intention is to have ethereum.org point to the Eth1.0 specs repo, or vice versa. Where does the canonical content live? [6:14](https://youtu.be/zz1IhYKztuc?t=374)
```

> **ACTION**: (mainly James) Work out what content we think we should be surfacing on the Eth1.0 spec repo and network upgrades content for Ethtereum.org, and then speak to Thomas and Sam for their views. [18:28](https://youtu.be/zz1IhYKztuc?t=1108)

Reviewer: ```suggestion
**ACTION 25.2**: (mainly James) Work out what content we think we should be surfacing on the Eth1.0 spec repo and network upgrades content for Ethtereum.org, and then speak to Thomas and Sam for their views. [18:28](https://youtu.be/zz1IhYKztuc?t=1108)
```

> **ACTION**: Cat Herders to consider EIP information for Ethereum.org site. [19:48](https://youtu.be/zz1IhYKztuc?t=1188)

Reviewer: ```suggestion
**ACTION 25.3**: Cat Herders to consider EIP information for Ethereum.org site. [19:48](https://youtu.be/zz1IhYKztuc?t=1188)
```

> **ACTION**: Brent to continue investigation into sources of EIPs information. [31:36](https://youtu.be/zz1IhYKztuc?t=1896)

Reviewer: ```suggestion
**ACTION 25.4 **: Brent to continue the investigation into sources of EIPs information. [31:36](https://youtu.be/zz1IhYKztuc?t=1896)
```

> **ACTION**: Cat Herders to progress EIP status cleanup. [34:12](https://youtu.be/zz1IhYKztuc?t=2051)

Reviewer: ```suggestion
**ACTION 25.5**: Cat Herders to progress EIP status cleanup. [34:12](https://youtu.be/zz1IhYKztuc?t=2051)
```

**Brent** - Did we lose Micah?

**Hudson** - Yes,we lost Micah. I think the summary of what Micah was saying... unless, Micah you back?

**Micah** - Where did I cut off there?

**Hudson** - You were talking about what was missing or the the fact that there were definitely some things that were missing and there were articles that were written that provided supplemental information that could need to go somewhere.

**Micah** - Yeah that's basically it. We've got eip 1559, a great example of this. We've got tons of supplemental information and nowhere to put it and so we can link people to it of course when we have a channel and we have people coming in and talking to it but there's no place that people can know to go to read more about any supplemental information about an eip because we want to keep that out of the eip itself because the eip is a technical document. But we don't have somewhere else to send the person to say "hey if you ever want to know more, like opinion stuff on an eip, go here."

**Hudson** - Ah okay 

> **Pooja** - So that is the thing that we are proposing here. I'm gonna share the link in the chat - this is the place i mean. On the ethereum cat header website we are trying to curate a list of resources for any eip under discussion. For example right now we have three or four eips;  1559, 2938, evm384 and all that stuff, so whatever resources other than the eip at magicians forum, like blogs published by you and others, we curate it and keep it there. The idea here is that we can make it publicly available with the help of ethereum.org. They are thinking in this direction and looking for some thoughts like if this is something good that we should be doing.

Reviewer: ```suggestion
**Pooja** - So that is the thing that we are proposing here. I'm gonna share the [link in the chat](https://github.com/ethereum-cat-herders/ethereum-cat-herders.github.io/blob/master/index.md#1559-fee-market-change). On the ethereum cat header website we are trying to curate a list of resources for any eip under discussion. For example right now we have three or four eips;  1559, 2938, evm384 and all that stuff, so whatever resources other than the eip at magicians forum, like blogs published by you and others, we curate it and keep it there. The idea here is that we can make it publicly available with the help of ethereum.org. They are thinking in this direction and looking for some thoughts like if this is something good that we should be doing.
```

**Hudson** - I think this is a good idea. I think that there's a bigger issue around funding that'll be a completely different topic in today's meeting, if we have time, or a different one. But I think this is an important enough change that we should ask the cat herders for funding for it now that their treasury has gotten bigger from the price increase, especially if we're going to be paying out an eth or something like that. But we'll have to talk to the rest of the cat herders about that so that should be on the agenda for their meeting and then if that is approved for funding from the cat herders we could do git coin. I'd prefer git coin but we don't have to use git coin for it necessarily.

**Micah** - Today i learned the cat herders have money!

**Hudson** - Cool 

**Pooja** - Yeah but I believe getting an estimate of what amount of funding or the effort that we are looking for here would help us. 

**Hudson**  - Yeah that would be helpful for a cat herders decision. Micah do you have any ideas or anybody else in the call?

> **Micah** - Eestimating software is the hardest job in the world. So relative to building an ethereum client this is a trivial task. This is one of those things like it depends on the skill level you hire and whether the person already has at least some knowledge of how this works. If you were to hire lightclient or me it would be an easier job than if you've hired some stranger but i think lightclient and me both are going to be more expensive and have little time available. If anyone wants to interrupt me please do. 

Reviewer: ```suggestion
**Micah** - Estimating software is the hardest job in the world. So relative to building an ethereum client this is a trivial task. This is one of those things like it depends on the skill level you hire and whether the person already has at least some knowledge of how this works. If you were to hire lightclient or me it would be an easier job than if you've hired some stranger but i think lightclient and me both are going to be more expensive and have little time available. If anyone wants to interrupt me please do. 
```

**Pooja** - I think that is the current situation with ethereum.org/eips that it is just an overview page and it actually points towards the eips.ethereum.com?

**Brent** - Okay i'll look into that some more. I gotta find some more time to work on some of this stuff so still working on it.

# 4. EIP status change (update)

**Pooja** - Okay, so moving on the next item is eip status change update. The cat herders have been working with eip authors for the proposals which are in a draft status or maybe not in the correct status even if it is in like in review. In the last call it is more than a month or so so we are trying to reach out to those authors and asking them to change their status. So we made some progress i think. At least three of the eips we closed last week. By close i mean that it wasn't draft status, now they are in review. For berlin eip i think there are three eips which are still in draft and that they that they need to move into review status. If there is no change done by the author this week  we will be creating a pull request to change it to review. The eips are eip2718 eip2929 and eip2315. All these three are in draft so we would like to have that in review.

> Other than that with the help of p penny series we interacted with quite a few authors and we suggested them to make a request to move on their eip from draft to review but there are quite a few which haven't created any pull requeste yet. If they are not done by the end of next week cat headers will be reaching out to them and try to create a pull request to change the status.

Reviewer: ```suggestion
Other than that with the help of Peep an EIP series, we interacted with quite a few authors and we suggested them to make a request to move on their eip from draft to review but there are quite a few which haven't created any pull request yet. If they are not done by the end of next week cat headers will be reaching out to them and try to create a pull request to change the status.
```
